prac_06/programming_language.py

```diff
@@ -0,0 +1,21 @@
+"""CP1404/CP5632 Practical
+Estimated time: 20min"""
+
+
+class ProgrammingLanguage:
+    """Represents a programming language object"""
+
+    def __init__(self, name="", is_dynamic=True, is_reflection=True, year=0):
+        """"""
+        self.name = name
+        self.is_dynamic = is_dynamic
+        self.is_reflection = is_reflection
+        self.year = year
+
+    def __str__(self):
+        """"""
```

Comment:
Empty docstrings wouldn't show any warnings. But, it is better to include function's purpose in it